lib/mm_network.pm

```diff
@@ -128,7 +128,7 @@ sub configure_static_dns {
     my $servers = join(" ", @{$conf->{nameserver}});
 
     if ($is_nm) {
-        $nm_id = script_output('nmcli -t -f NAME c | head -n 1') unless ($nm_id);
+        $nm_id = script_output('nmcli -t -f NAME c | grep -v ^lo: | head -n 1') unless ($nm_id);
```

Comment:
you could even do that on the perl side, but we can leave that for a follow up ticket, same for the ones earlier